build/config.mak

```
@@ -4,7 +4,7 @@ PLATFORM ?= device
 DEBUG ?= 0
 
 HOME_DISPLAY_EXTERNALS ?= 1
-EPSILON_VERSION ?= 15.5.0
+EPSILON_VERSION ?= 16.4.0 #Default: 15.5.0
```

No, we use epsilon 15.5.0
```suggestion
EPSILON_VERSION ?= 15.5.0
```